lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -196,6 +196,9 @@ def add_payment_method(doc, payment_method)
             doc.expDate(exp_date(payment_method))
             doc.cardValidationNum(payment_method.verification_value)
           end
+          doc.cardholderAuthentication do
```

Comment:
Should we wrap the whole `cardholderAuthentication` block in the if statement so it doesn't even appear unless we have a cryptogram? Otherwise there will always be an empty element.